packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -157,11 +154,9 @@ _ref:
                 blocks:
                   - id: content_card
                     type: Card
+                    properties:
```

Comment:
In the example description we say we are going to add a title block. We should probably just adjust that sentance

Reply:
I have changed up the example description. Let me know what you think and if it can be improved or should be worded differently.
Change made in 386463869c78c494c05636e1e435419c96917b02